R/surv-roc_survival_curve.R

```diff
@@ -7,7 +7,7 @@
 #'
 #' @return
 #' A tibble with class `roc_survival_df`, `grouped_roc_survival_df` having
-#' columns `.threshold`, `recall`, and `precision`.
+#' columns `.threshold`, `recall`, `precision`, and `.eval_time`.
```

Comment:
I think that this is a copy/paste error (or it is being computed completely wrong)

Reply:
done in [`0153bb4` (#407)](https://github.com/tidymodels/yardstick/pull/407/commits/0153bb437cf3b688eafb7f6636ac4256c781cbf8)